content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,219 @@
+records that match **any** of the rules contained inside. This uses the same logic as an OR (|)
+logical operator.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Group by
+~~~~~~~~
+
+Once all filters are set, click the :guilabel:`Add` button to add these filters to the search bar.
+To have the report grouped appropriately, click the :icon:`fa-caret-down` :guilabel:`(down caret)`
+icon, to the right of the search bar, and click :guilabel:`Salesperson` in the :guilabel:`Group
+By` section. All results are now grouped by the salesperson assigned to each lead.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`crm/track_leads/essential-filters` in the order specified to achieve a heavily-detailed
+filter.
+
+- **Referred-by:** Filter for referrals, such as by appointment or sales member.
+
+- **Source:** Filter for specific source UTMs, such as Facebook or LinkedIn.
+
+- **Notes:** Filter for internal notes.
+
+- **Tags:** Filter for categorical tags.
+
+- **Email:** Filter for specific email domains, such as gmail.com or yahoo.com.
+
+- **Salesperson:** Filter for leads associated with certain sales members.
+
+These conditions can be added, removed, or modified to best fit the desired information in
+the report.
+
+.. seealso::
+  `Additional rules for quality leads <https://www.odoo.com/documentation/17.0/applications/sales/
+  crm/track_leads/quality_leads_report.html#add-rules-for-quality-leads>`_
+
+.. tip::
+
+  Once the rules for the filter are set, click the purple :guilabel:`Confirm` button at the
+  bottom of the pop-up menu to save the custom filter and close the pop-up menu.
+
+  .. image:: lead_distribution_report/confirm-filter.png
+    :align: center
+    :alt: Click the confirm button to save the filter.
+
+
+  The :guilabel:`Pipeline Analysis` dashboard is now displayed again with each filter rule in the
+  search bar. Click the :icon:`fa-caret-down` :guilabel:`(down caret)` icon, to the
```

Comment:
Lines 186-195 might make for a better conclusion outside of the **Tip** admonition block. The "save current search" content is the actual **Tip**